### PR TITLE
Ns 5

### PR DESCRIPTION
**Pull request by Poostaq**
Finished specialization - please review and test

### CODE REVIEW

File: game/DataStore/databaseOperations.gd [hunk 1]
```diff
@@ -201,3 +191,46 @@ func _datetime_to_string(date):
...
+	var select = "SELECT specialization_identifier "
+	var from = "FROM specializations "
+	var where = "WHERE specialization_enum is not null"
+	var selected_array = _sql_select(select+from+where)
+	print(selected_array)
+	db.close_db()
+	return selected_array
+
+
+func read_data_for_specialisation(specialisation_identifier):
```
Comment (Malynna) [inline game/DataStore/databaseOperations.gd:206]: specialization

File: game/DataStore/databaseOperations.gd [hunk 2]
```diff
@@ -201,3 +191,46 @@ func _datetime_to_string(date):
 		})
 	else :
 		print ("Bad sysdate")
+
+
+func read_specialisation_identifiers():
```
Comment (Malynna) [inline game/DataStore/databaseOperations.gd:196]: specialization

File: game/SpecialisationPage/SpecialisationStep.gd [hunk 3]
```diff
@@ -36,14 +40,52 @@ func _ready() -> void:
...
+	else:
+		_current_specialisation_index -= 1
+	load_step()
+
+func _on_NextSpecialisation_button_up():
+	if _current_specialisation_index == len(_specialisation_list)-1:
+		_current_specialisation_index = 0
+	else:
+		_current_specialisation_index += 1
+	load_step()
```
Comment (Malynna) [inline game/SpecialisationPage/SpecialisationStep.gd:91]: specialization not specialisation (in file name too)

File: game/SpecialisationPage/ChooseSpecialisationStep.tscn [hunk 4]
```diff
@@ -0,0 +1,359 @@
...
+
+[sub_resource type="DynamicFont" id=2]
+size = 20
+font_data = ExtResource( 6 )
+
+[sub_resource type="DynamicFont" id=4]
+size = 40
+font_data = ExtResource( 6 )
+
+[node name="SpecialisationStep" type="Control"]
```
Comment (Malynna) [inline game/SpecialisationPage/ChooseSpecialisationStep.tscn:31]: specialization not specialisation(file name and all rows)

**Review by Malynna — CHANGES_REQUESTED** [review]
specialization not specialisation (all rows and file names)

File: game/SpecialisationPage/ChooseSpecialisationStep.tscn (hunk 4 above)
Comment (Poostaq) [reply game/SpecialisationPage/ChooseSpecialisationStep.tscn]: solved

File: game/SpecialisationPage/SpecialisationStep.gd (hunk 3 above)
Comment (Poostaq) [reply game/SpecialisationPage/SpecialisationStep.gd]: solved

File: game/DataStore/databaseOperations.gd (hunk 2 above)
Comment (Poostaq) [reply game/DataStore/databaseOperations.gd]: solved

File: game/DataStore/databaseOperations.gd (hunk 1 above)
Comment (Poostaq) [reply game/DataStore/databaseOperations.gd]: solved

**Pull request merged**